packages/mdl-radio/event-bus.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Class to coordinate between between components without strict dependencies.
+ */
+export default class EventBus {
```

Comment:
This is a good solve, but unfortunately I think it may be a bit to "framework"-y for us. As soon as we start getting into things like defining our own event handling mechanisms, we start stepping on the toes of frameworks.

If I'm understanding the code correctly, it looks like this is primarily used to coordinate between radio instances such that when a radio button in a group becomes checked, the others become unchecked. I think an approach similar to that taken for [checkbox](https://github.com/google/material-design-lite/blob/f3c1428/packages/mdl-checkbox/mdl-checkbox.scss#L155) - where we use adjacent sibling combinators rather than explicit classes to style the components - would solve this issue for us without the need for any JS coordination between the components. What do you think of that approach?

If we do decide to keep this however, we should probably be using node's built-in [EventEmitter](https://nodejs.org/api/events.html) (which webpack can bundle) instead of rolling our own.